clientd3d/d3drender.c

```diff
@@ -186,6 +181,26 @@ void SetZBias(LPDIRECT3DDEVICE9 device, int z_bias) {
                                    *((DWORD *) &bias));
 }
 
+bool isFogEnabled()
+{
+	return gD3DDriverProfile.bFogEnable;
+}
+
+long getShadeAmount()

```

Comment:
If there's a getter for a variable, then the variable should be declared static in that file, so that the getter is the only way to access it.  Also check sun_vect and gWireframe.